Traceback (most recent call last):
  File "/home/mhostert/DarkNews/MiniBooNE-DarkNews/SymmetryFit/fit_3p2_general.py", line 198, in <module>
    fit_point_3p2(
  File "/home/mhostert/DarkNews/MiniBooNE-DarkNews/SymmetryFit/fit_3p2_general.py", line 164, in fit_point_3p2
    gf.save_to_locked_file(
  File "/home/mhostert/DarkNews/MiniBooNE-DarkNews/src/fastbnb/grid_fit.py", line 38, in save_to_locked_file
    os.remove(lockfile_name)
FileNotFoundError: [Errno 2] No such file or directory: 'results/3p2_general_mzprime_m4_0.015_eps_0.0001/5m_13d_8h_34m_46s_invmass_spectrum.dat.lock'
srun: error: cn014: task 0: Exited with exit code 1
